app/pdf-viewer/buffer.py

```diff
@@ -243,6 +243,11 @@ def __init__(self, url, config_dir, background_color, buffer_id, emacs_var_dict)
              self.emacs_var_dict["eaf-emacs-theme-mode"] == "dark")):
             self.inverted_mode = True
 
+        # Inverted mode exclude image.
+        self.inverted_mode_exclude_image = True
+        if (self.emacs_var_dict["eaf-pdf-dark-exclude-image"] == "false"):
+            self.inverted_mode_exclude_image = False
```

Comment:
self.inverted_mode_exclude_image = self.emacs_var_dict["eaf-pdf-dark-exclue-image"] == "true" 

Is better way.

Anyway I have fix this problem in newest version.